site/blocks/front-matter.scss

```diff
@@ -0,0 +1,103 @@
+.front-matter {
```

Comment:
nit: I haven't been super consistent with this but I tried to prefix WP blocks with `wp-block-owid`, e.g. [wp-block-owid-prominent-link](https://github.com/owid/owid-grapher/blob/55fbca9d07bde041c91b95ef379f4b4cd1d79f1e/site/blocks/prominent-link.scss#L1), to avoid possible style collisions.

I also find the familiar prefix helpful when navigating multiple layers of rendering code.

Reply:
> The color discrepancy makes for a confusing signal: maybe best to keep the button state-free?

Oops, this was an oversight on my `:visited` QA. 

Button color is now consistent. 👍

Reply:
Seeing as we'll eventually be deprecating WP I'm not sure it makes much of a difference at this point.

I'm hoping that we'll be able to have universal implementations for various aspects of the design (typography, components, etc) in which we may be able to re-establish all of these conventions consistently and correctly.